### PR TITLE
MTK - 55.7.0 release branch 

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Added release notes as per MTK-1269

### CODE REVIEW

**Review by mlewandowski-edb — CHANGES_REQUESTED** [review]
The changes currently in the PR look good to me; however, the link lable in the Windows section of https://www.enterprisedb.com/docs/migration_toolkit/latest/installing/ page needs to be updated to include:

- Windows Server 2022
- Windows 11

**Review by mlewandowski-edb — APPROVED** [review]

File: product_docs/docs/migration_toolkit/55/02_supported_operating_systems_and_database_versions.mdx [hunk 1]
```diff
@@ -25,10 +25,19 @@ You can use the following database product versions with Migration Toolkit:
 -   (Sybase) SAP Adaptive Server Enterprise 15.7
 -   (Sybase) SAP Adaptive Server Enterprise 16.0
 
+!!! Note
+All the PostgreSQL and EDB Postgres Advanced Server versions available as BigAnimal single-node, primary/standby high-availability, and distributed high-availability cluster types are also supported for use with Migration Toolkit. See the BigAnimal (EDB’s managed database cloud service) [documentation](/biganimal/latest) for more information about BigAnimal’s [supported cluster types](/biganimal/latest/overview/02_high_availability/). See the [database version policy documentation](/biganimal/latest/overview/05_database_version_policy/) for the versions of PostgreSQL and EDB Postgres Advanced Server available in BigAnimal.
+
+The superuser role isn't available in BigAnimal EDB Postgres Advanced Server distributed high-availability cluster databases. As a result, the following Migration Toolkit capabilities that require superuser access aren't supported when targeting BigAnimal distributed high-availability clusters:
+- the `-copyViaDBLinkOra` option that enables data migration using db_link_ora extension
```
Comment (ebgitelman) [inline product_docs/docs/migration_toolkit/55/02_supported_operating_systems_and_database_versions.mdx:32]:
```suggestion
- The `-copyViaDBLinkOra` option that enables data migration using the db_link_ora extension
```

File: product_docs/docs/migration_toolkit/55/02_supported_operating_systems_and_database_versions.mdx [hunk 2]
```diff
@@ -25,10 +25,19 @@ You can use the following database product versions with Migration Toolkit:
...
+- the Oracle compatible SQL command `CREATE DATABASE LINK`
```
Comment (ebgitelman) [inline product_docs/docs/migration_toolkit/55/02_supported_operating_systems_and_database_versions.mdx:33]:
```suggestion
- The Oracle-compatible SQL command `CREATE DATABASE LINK`
```

File: product_docs/docs/migration_toolkit/55/02_supported_operating_systems_and_database_versions.mdx [hunk 3]
```diff
@@ -25,10 +25,19 @@ You can use the following database product versions with Migration Toolkit:
...
+!!!
+
 Contact your EnterpriseDB Account Manager or [sales@enterprisedb.com](mailto:sales@enterprisedb.com) if you require support for other database products.
 
 !!! Note
-    The Migration Toolkit isn't tested with and doesn't officially support use with Oracle Real Application Clusters (RAC) and Exadata. However, Migration Toolkit might work when it is connected to a single persistent node. For more information, contact your EDB representative.
+The Migration Toolkit isn't tested with and doesn't officially support use with Oracle Real Application Clusters (RAC) and Exadata. However, Migration Toolkit might work when it is connected to a single persistent node. For more information, contact your EDB representative.
```
Comment (ebgitelman) [inline product_docs/docs/migration_toolkit/55/02_supported_operating_systems_and_database_versions.mdx:39]:
```suggestion
The Migration Toolkit isn't tested with and doesn't officially support use with Oracle Real Application Clusters (RAC) and Exadata. However, Migration Toolkit might work when it's connected to a single persistent node. For more information, contact your EDB representative.
```

File: product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+navTitle: Version 55.7.0
+---
+
+Released: 12 Dec 2023
+
+New features, enhancements, bug fixes, and other changes in Migration Toolkit 55.7.0 include:
+
+|     Type     |                                                                                                                                                                                         Description                                                                                                                                                                                         |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Enhancement  | The Migration Toolkit is now certified to work with all the BigAnimal supported cluster types including the distributed high availability clusters. |
```
Comment (ebgitelman) [inline product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx:12]:
```suggestion
| Enhancement  | The Migration Toolkit is now certified to work with all the BigAnimal supported cluster types including the distributed high-availability clusters. |
```

File: product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+| Enhancement  | Improved the performance of table-level parallel loading of many tables by updating the operation to fetch all the constraints in a single network round trip. [Support ticket #97379] |
+| Bug&nbsp;fix | Fixed an issue where user created triggers on the target PostgreSQL or EDB Postgres Advanced Server databases were getting disabled during the data migration phase. [Support ticket #97637] |
```
Comment (ebgitelman) [inline product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx:14]:
```suggestion
| Bug&nbsp;fix | Fixed an issue where user-created triggers on the target PostgreSQL or EDB Postgres Advanced Server databases were getting disabled during the data migration phase. [Support ticket #97637] |
```

File: product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+| Bug&nbsp;fix | Fixed an issue that caused re-evaluation of the same table with an indefinite loop for table-level parallel loading mode. [Support ticket #96054]|
```
Comment (ebgitelman) [inline product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx:15]:
```suggestion
| Bug&nbsp;fix | Fixed an issue that caused reevaluation of the same table with an indefinite loop for table-level parallel loading mode. [Support ticket #96054]|
```

File: product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+| Bug&nbsp;fix | Fixed an issue where data copy was failing while migrating from a source Oracle database configured with the Latin9 character set. [Support ticket #99475, #99506] |
+| Bug&nbsp;fix | Fixed an issue where the copy API was failing to copy the data and a subsequent attempt to copy the data with the bulk insert mode was failing with a duplicate error in the target database.[Support ticket #99506, #97784] |
+| Bug&nbsp;fix | Fixed an issue where a data copy was failing while migrating a source SQL Server table with a case sensitive column name in the EDB Postgres Advanced server target database. |
```
Comment (ebgitelman) [inline product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx:18]:
```suggestion
| Bug&nbsp;fix | Fixed an issue where a data copy was failing while migrating a source SQL server table with a case-sensitive column name in the EDB Postgres Advanced server target database. |
```

File: product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+| Bug&nbsp;fix | Fixed an issue where a data copy was failing while migrating a source MySQL table due to a case sensitive table name. |
```
Comment (ebgitelman) [inline product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx:19]:
```suggestion
| Bug&nbsp;fix | Fixed an issue where a data copy was failing while migrating a source MySQL table due to a case-sensitive table name. |
```

File: product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+| Bug&nbsp;fix | Fixed an issue for partitioned tables with a SUBPARTITION TEMPLATE clause that caused a partitioned table sub-partitioned by range to fail to migrate. |
+| Bug&nbsp;fix | Fixed an issue where the ``capture_admin`` database system group was getting included when the ``-allgroups`` option was specified for migration from the source EDB Postgres Advanced Server version 16 database. |
```
Comment (ebgitelman) [inline product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx:21]:
```suggestion
| Bug&nbsp;fix | Fixed an issue where the `capture_admin` database system group was getting included when the `-allgroups` option was specified for migration from the source EDB Postgres Advanced Server version 16 database. |
```

File: product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+| Bug&nbsp;fix | Fixed an issue that caused re-evaluation of the same table with an indefinite loop for table-level parallel loading mode. [Support ticket #96054]|
+| Bug&nbsp;fix | Fixed an issue where data copy was failing while migrating from a source Oracle database configured with the Latin9 character set. [Support ticket #99475, #99506] |
+| Bug&nbsp;fix | Fixed an issue where the copy API was failing to copy the data and a subsequent attempt to copy the data with the bulk insert mode was failing with a duplicate error in the target database.[Support ticket #99506, #97784] |
+| Bug&nbsp;fix | Fixed an issue where a data copy was failing while migrating a source SQL Server table with a case sensitive column name in the EDB Postgres Advanced server target database. |
+| Bug&nbsp;fix | Fixed an issue where a data copy was failing while migrating a source MySQL table due to a case sensitive table name. |
+| Bug&nbsp;fix | Fixed an issue for partitioned tables with a SUBPARTITION TEMPLATE clause that caused a partitioned table sub-partitioned by range to fail to migrate. |
```
Comment (ebgitelman) [inline product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx:20]:
```suggestion
| Bug&nbsp;fix | Fixed an issue for partitioned tables with a `SUBPARTITION TEMPLATE` clause that caused a partitioned table sub-partitioned by range to fail to migrate. |
```

**Review by mlewandowski-edb — DISMISSED** [review]

**Review by mlewandowski-edb — DISMISSED** [review]

File: product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx (hunk 4 above)
Comment (nidhibhammar) [reply product_docs/docs/migration_toolkit/55/mtk_rel_notes/mtk_5570_rel_notes.mdx:12]: ok @mlewandowski-edb.

**Review by djw-m — APPROVED** [review]
LGTM (Tho am reviewing during a talk at pgconf.eu :) )

**Pull request merged**